dapp/src/lib/utils/getGasSummary.ts

```diff
@@ -34,6 +34,9 @@ export function getGasSummary(
         totalGas: totalGas?.toString(),
         isSponsored,
         gasUsed: effects.gasUsed,
+        gas() {
+            return BigInt(this.totalGas || 0);
+        },
```

Comment:
```suggestion
        get gas() {
            return BigInt(this.totalGas || 0);
        },
```
so you can use it like `gasSummary.gas` and no need to call it